File: Capitulo05-Exercicio06.c
// ADS 2020 - LINGUAGEM DE PROGRAMAÇÃO
// CAPÍTULO 05 - EXERCÍCIO 04
/* Criar em linguagem C e utilizando o comando DO WHILE, que imprima a palavra FATEC o número
de vezes que for digitado em um intervalo de um segundo.*/

#include<stdio.h>
#include<stdlib.h>
#include<conio.h>
#include<locale.h>
#include<time.h>

main()
{
    setlocale(LC_ALL,""); // Configura para o portugues do Brasil
    system("cls");
    int n; // Numero de vezes da impressao
    printf("\nCAPÍTULO 05 - EXERCÍCIO 04\n");
    printf("\nDigite a quantidade de vezes que deseja imprimir FATEC: ");
    scanf("%d", &n);
    printf("\n");
    int i = 1; // Controle do laco
    while(n>=i)
    {
        if(((i%5)==0)&&(i!=0))printf(" ... FATEC ...\n");
        else printf(" ... FATEC");
        sleep(1000);
        i++;
    }
    if(((i%5)==0)&&(i!=0))printf("\n");
    else printf(" ... \n");    
    printf("\nAte mais!!!");
    sleep(2000);
}

File: Capitulo05-Exercicio06.py
# ADS 2020 - LINGUAGEM DE PROGRAMAÇÃOO
# CAPÍTULO 05 - EXERCÍCIO 06
''' Criar em linguagem  e utilizando o comando DO WHILE, que imprima a palavra FATEC o número
de vezes que for digitado em um intervalo de um segundo.'''

from pynput.keyboard import key, Controller
from time import sleep

print("CAPÍTULO 05 - EXERCÍCIO 06")
print("")
num = int(input("Digite a quantidade de vezes que deseja imprimir FATEC: "))
c = 1
n = num
print("\nUtilizando WHILE")
while c <= num:
    print("FATEC, ", end="")
    c = c + 1
    sleep(1)
n = num
print("")
print("\n Utilizando FOR")
for n in range(0, n):
    print("FATEC, ", end="")
    sleep(0.5)
    n -= 1
print("")
print("\nPrograma finalizado.")